scylla/src/transport/locator/replicas.rs

```diff
@@ -8,7 +8,7 @@ pub type NodeRef<'a> = &'a Arc<Node>;
 /// This type is very similar to `Cow`, but unlike the previously mentioned it holds references in a
 /// `Owned` variant `Vec`.
 #[derive(Debug)]
-pub enum Replicas<'a> {
+pub(crate) enum Replicas<'a> {
```

Comment:
No need to do it there, `replicas` (and other child modules of `locator`) can be declared private.